src/http/AuthHttpClient.js

```diff
@@ -1172,35 +1173,35 @@ class AuthHttpClient {
     async getAliases() {
         const request = {
             method: 'get',
-            url: '/aliases'
+            url: '/aliases',
         };
         return this._instance(request);
     }
 
     /**
      * Get default recovery agent.
-     * @return {Object} GetDefaultAgentResponse proto buffer
+     * @return {Object} GetDefaultAgentResponse gen buffer
```

Comment:
why `gen` instead of proto?

Reply:
webstorm auto refactored this when I moved proto file's path  🙁